src/main/java/com/google/devtools/build/lib/rules/java/JavaModuleFlagsProvider.java

```diff
@@ -42,7 +42,7 @@
  * targets.
  */
 @AutoValue
-abstract class JavaModuleFlagsProvider
+public abstract class JavaModuleFlagsProvider
```

Comment:
Is this visibility change only for testing? If so, would it be possible to keep this package-private and make the tests use the `JavaInfo#getJavaModuleFlagsInfo()` Starlark API?

Reply:
Fixed